source/dhtslib/vcf/reader.d

```diff
@@ -175,7 +175,8 @@ struct VCFReaderImpl(CoordSystem cs, bool isTabixFile)
         // * unpacking and
         // * destroying
         // its copy
-        return VCFRecord(this.vcfhdr, bcf_dup(this.b), this.MAX_UNPACK);
+        auto copiedBcf_1t = bcf_dup(this.b);
+        return VCFRecord(this.vcfhdr, copiedBcf_1t, this.MAX_UNPACK);
     }

```

Comment:
I think this is the only line I had to change to get `-dip1000` to work for the rest of the package. But most of dhtslib code is not annotated as `@safe`. Not sure if that affects the compiler's ability to do checking for `scope`.

Reply:
Make sure to add a separate CI compilation that uses `-dip1000` as Nic suggests

Reply:
Nvm, I see you did